mithril-client/src/commands/download.rs

```diff
@@ -36,8 +41,18 @@ impl DownloadCommand {
             .map_err(|e| format!("configuration deserialize error: {e}"))?;
         debug!("{:?}", config);
         let runtime = Runtime::new(config.network.clone());
-        let aggregator_handler =
-            AggregatorHTTPClient::new(config.network.clone(), config.aggregator_endpoint);
+        // TODO: This does not allow the client to handle an upgraded API version of a new supported era after the switch
```

Comment:
This piece of code is repeated in each client's following files, maybe it is possible to refactor this in a function.

Reply:
Done :heavy_check_mark: